compiler/lib/code.ml

```diff
@@ -152,7 +154,8 @@ end = struct
 
   let to_string ?origin i = Var_printer.to_string printer ?origin i
 
-  let print f x = Format.fprintf f "v%d" x
+  (*let print f x = Format.fprintf f "v%d" x*)
+  let print f x = Format.fprintf f "%s" (to_string x)
```

Comment:
What is this diff about ?

Reply:
It was to clarify variable names in debugging. I had forgotten to remove it.